src/content/blog/2025-05-05-celebrating-ten-years-of-discover.mdx

```diff
@@ -0,0 +1,68 @@
+
+## Challenges to Overcome
+
+UGRC's new service worked the first time without a hitch and nothing remotely bad ever happened. There were so few bugs in the system, it may as well have been the pristine, barren landscape of Antarctica. There wasn't even one user complaint…said no developer ever, right? Like any new technology, there were more than a few hurdles to jump over in order to create a streamlined, modernized server for Utah. Many of those hurdles had to do with two unique aspects of Discover: serving the millions of pre-cached cloud-based tiles, and user access.
+
+When you view services through Discover, you may think you are looking at a one really big image of Utah. But, this image of Utah is actually composed of millions of image 'tiles', all mosaicked together into a series (just like in a real mosaic floor or ceiling, it takes many colorful tiles to make up the whole artwork). These tiles are pre-cached to increase rendering speed and allows for seamless transitions as you pan or zoom between them. They render so quickly, that many users don't even realize there is more than one tile to begin with.
+
+However, like ragtag siblings stuck in their “get-along” shirt, some of these tiles didn't quite play nice at first. Some of them were so-called 'sticky tiles': when the user zoomed in or out, the other nearby tiles would zoom in or out as well, but sticky tiles would be stuck at the previous zoom level as though glued in place. Other problem children included 'delinquent tiles', which were tiles that didn't show up in the service at all. Apparently, those tiles had somewhere better to be. And many of the other perfectly innocent tiles were hampered by slow rendering speeds.
+
+Luckily, through our team's many long hours of dedicated troubleshooting, these issues were resolved and new solutions were implemented. Nowadays, all the tiles sit amicably at the dinner table and do as they're told, which is to provide a seamless user experience for those who rely on aerial imagery and basemaps to do their jobs. The Discover services available today are much more robust than those from the old ArcGIS Image Server. It is cloud-based, utilizes Node.js and a Postgres database with all usage metrics stored in Google BigQuery, and is built to last.
+
+## The Enduring Impact of Publicly Available Imagery
+
+As of the writing of this article, Discover has seen over 5.4 billion requests over its lifetime. Each of these requests represents a pan or zoom made by a wide variety of users, from government agencies to engineering firms to data managers. These entities use the services for all kinds of workflows, including many that impact people in Utah every single day.
+
+For example, the [Utah Division of Emergency Management](https://dem.utah.gov/) uses aerial imagery to safeguard Utah's future and help mitigate the effects of natural disasters in our state. In their words:
+
```

Comment:
These quotes may stand out more visually if we do block quotes (add a `>` at the beginning of each line). My first time reading through it kind of threw me for a loop when the voice changed, I didn't realize we were in a quote.

Reply:
done